client/my-sites/plans/plan-overview/plan-features/feature.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+import SectionHeader from 'components/section-header';
+
+const PlanFeature = React.createClass( {
```

Comment:
You can use the syntax for a functional stateless component here, if you want:

```js
const PlanFeature = ( { button, description, heading } ) => {
  return (
    <CompactCard className="plan-features__feature">
      { ... }
    </CompactCard>
  );
} );

PlanFeature.propTypes = {
  button: React.PropTypes.shape( {
    label: React.PropTypes.string.isRequired,
    onClick: React.PropTypes.func.isRequired
  } ),
  description: React.PropTypes.string.isRequired,
  heading: React.PropTypes.string.isRequired
};
```

This makes it a bit more obvious that this is a component that isn't meant to need state or any React lifecycle methods.